gwswlib/importer.py

```diff
@@ -25,24 +26,43 @@ def import_hydx(hydx_path):
         "Verbinding1.csv",
         "Verloop1.csv",
     ]
+    connectedcsvfiles = [
+        # "BOP1.csv",
+        # "Debiet1.csv",
+        # "GroeneDaken1.csv",
+        # "ItObject1.csv",
+        "Knooppunt1.csv",
+        "Kunstwerk1.csv",
+        # "Meta1.csv",
+        # "Nwrw.csv",
+        # "Oppervlak1.csv",
+        # "Profiel1.csv",
+        "Verbinding1.csv",
+        # "Verloop1.csv",
+    ]
 
-    # check if csv file exists
-    for f in csvfiles:
+    existing_files = []
+    for f in hydxcsvfiles:
         csvpath = os.path.join(hydx_path, f)
```

Comment:
Klein trucje: als je er `os.path.abspath()` omheen zet krijg je een absoluut path. Voor een foutmelding kan dat lekker zijn om aan te geven van waar uit het script loopt te zoeken.
Vaak is het niet nodig, hoor. Soms hoeft het ook alleen in de logging.
Ter info mocht dit wèl een punt zijn waarop het handig is.